tests/integration/cloudformation/resource_providers/ssm/test_parameter.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from localstack.aws.connect import ServiceLevelClientFactory
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest.fixtures import StackDeployError
+
+RESOURCE_GETATT_TARGETS = [
+    "Type",
+    "Description",
+    "Policies",
+    "AllowedPattern",
+    "Tier",
+    "Value",
+    "DataType",
+    "Id",
+    "Name",
+]
+
+
+class TestAttributeAccess:
+    @pytest.mark.parametrize("attribute", RESOURCE_GETATT_TARGETS)
+    @pytest.mark.xfail(
+        reason="Some tests are expected to fail, since they try to access invalid CFn attributes"
+    )
+    @pytest.mark.skipif(condition=not is_aws_cloud(), reason="Exploratory test only")
+    def test_getattr(
+        self,
+        aws_client: ServiceLevelClientFactory,
+        deploy_cfn_template,
+        attribute,
+        template_root,
+        snapshot,
+    ):
+        """
+        Capture the behaviour of getting all available attributes of the model
+        """
+
+        try:
+            stack = deploy_cfn_template(
+                template_path=template_root.joinpath("resource_providers", "ssm", "parameter.yaml"),
```

Comment:
I'd prefer a version here where the main IDE in use (PyCharm) can jump to the file location. 
Fairly small thing but quite convenient when working with the tests, especially after the initial write during maintenance.